src/aggsig.rs

```diff
@@ -381,7 +380,7 @@ mod tests {
             let secnonce_2 = export_secnonce_single(&secp).unwrap();
 
             // Calculate public nonces
-            let pubnonce_1 = PublicKey::from_secret_key(&secp, &secnonce_1).unwrap();
```

Comment:
Maybe get rid of this completely if we don't use it anywhere?

Reply:
OK~